### ISSUE TITLE
Cannot call method 'substr' of undefined

### ISSUE THREAD

**Issue by NielsdeBlaauw** [apiwise/apiranto#11]
Trying to generate documentation for an API endpoint I have found it always fails on the same point, even though the definition file is valid swagger syntax.

The same error can also been seen generating documentation for the swagger [PetStore](http://petstore.swagger.io/v2/swagger.json) definition.

```
~/apiranto $ apiranto generate swagger.yml docs

/usr/local/lib/node_modules/apiranto/node_modules/jade/lib/runtime.js:240
  throw err;
        ^
TypeError: /usr/local/lib/node_modules/apiranto/templates/default.jade:71
    69|                         td(colspan=2)
    70|                           pre
  > 71|                             code.json!= helpers.renderJson(helpers.buildExample(response.schema))
    72| 

Cannot call method 'substr' of undefined
    at Object.highlight (/usr/local/lib/node_modules/apiranto/node_modules/highlight.js/lib/highlight.js:464:37)
    at Object.module.exports [as renderJson] (/usr/local/lib/node_modules/apiranto/src/helpers/renderJson.js:7:20)
    at $$l (eval at <anonymous> (/usr/local/lib/node_modules/apiranto/node_modules/jade/lib/index.js:218:8), <anonymous>:2153:68)
    at $$l (eval at <anonymous> (/usr/local/lib/node_modules/apiranto/node_modules/jade/lib/index.js:218:8), <anonymous>:2174:4)
    at $$l (eval at <anonymous> (/usr/local/lib/node_modules/apiranto/node_modules/jade/lib/index.js:218:8), <anonymous>:2196:4)
    at eval (eval at <anonymous> (/usr/local/lib/node_modules/apiranto/node_modules/jade/lib/index.js:218:8), <anonymous>:2203:4)
    at eval (eval at <anonymous> (/usr/local/lib/node_modules/apiranto/node_modules/jade/lib/index.js:218:8), <anonymous>:3899:4)
    at eval (eval at <anonymous> (/usr/local/lib/node_modules/apiranto/node_modules/jade/lib/index.js:218:8), <anonymous>:3960:22)
    at res (/usr/local/lib/node_modules/apiranto/node_modules/jade/lib/index.js:219:38)
    at Object.exports.renderFile (/usr/local/lib/node_modules/apiranto/node_modules/jade/lib/index.js:380:38)
```

**Comment by dvh** [comment]
Thanks for your feedback! We'll have a look at it.

**Comment by joostfarla** [comment]
The problem occurred when response attributes did not contain an "example" attribute. Fixed this in v0.0.4 by returning an arbitrary example value.
